package.json

```diff
@@ -14,7 +14,7 @@
     "precommit": "lint-staged"
   },
   "devDependencies": {
-    "eslint": "^4.4.1",
```

Comment:
This version is not very relevant to our users. You want to change the one in `packages/react-scripts/package.json` instead.